components/AddLostItem.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import styled from "styled-components";
+import { useState } from "react";
+import { lostItems } from "./Db";
+import { nanoid } from "nanoid";
+import Image from "next/image";
+
+export default function AddLostItem() {
+  const [description, setDescription] = useState("");
+  const [location, setLocation] = useState("");
+  const [name, setName] = useState("");
+  const [lost, setLost] = useState(lostItems);
+  const [img, setImage] = useState("/assets/MrBean.png");
+
```

Comment:
Hey Jannik, I think you can make it much easier for yourself by using just one state and update the different properties in the object. Or do you need these different states for something else? Then you don't need the onChange-functions and catch all the data when the form is submitted.